src/Accelerators/NNPA/Pass/DLCPasses.hpp

```diff
@@ -19,6 +19,9 @@
 
 namespace onnx_mlir {
 
+/// In ONNX, add pass for lowering Tensor types to MemRef types.
+std::unique_ptr<mlir::Pass> createONNXTensorToMemRefPass();
```

Comment:
This is not used anymore. I removed it in PR #1219.

Reply:
Could you please remove `createKrnlEmptyOptimizationPass` also? I think it is a dummy. Thanks!

Reply:
Will do thanks for pointing it out @tungld